js/SheetsApi.js

```diff
@@ -31,6 +31,14 @@ function SheetsApi() {
         return librarian;
     }
 
+    /** Public
+     * This function returns true if the current librarian is the administrator.
+     * @returns {boolean}
+     */
+    function isAdmin() {
+        return (librarian === "upei.personal.librarian@gmail.com");
```

Comment:
Rather than having the admin email hard-coded here, we need it in something like a configuration variable, like the keys, so a variable that is set near the top of this file, and then just the variable is used here.